### ISSUE TITLE
Radio mode (and search?) filter song by category

### ISSUE THREAD

**Issue by MeoMix** [MeoMix/StreamusChromeExtension#495]
It's possible to give a categoryId to the video list endpoint. We should retrieve the categoryId for music (which changes based on your local? maybe it's OK to just do English for now?) and filter results using that categoryId to improve result quality

https://developers.google.com/youtube/v3/docs/videoCategories/list
https://developers.google.com/apis-explorer/#p/youtube/v3/youtube.videos.list

**Comment by vairix-dmacchi** [comment]
mmm... maybe this could be good as an option... don't you think?

**Comment by MeoMix** [comment]
I don't think it should be an option, no. It's really bad when Streamus automatically suggests non-music via Radio Mode and is 100% never desirable.

It's possible that it could be an option when performing searches (or even just not apply it to search and let people see all results as expected), but for radio mode I am confident that we should be pruning the results more heavily.
